### ISSUE TITLE
Add compatibility with Django 3.0

### ISSUE THREAD

**Issue by claudep** [django-mptt/django-mptt#721]
Currently, mptt tests are failing with Django 3.0, apparently due to this Django commit:
https://github.com/django/django/commit/4122d9d3f198

**Comment by hlth** [comment]
Also breaking change is the removal of django utils.six and other python 2 compatibility APIs

https://docs.djangoproject.com/en/3.0/releases/3.0/#removed-private-python-2-compatibility-apis

**Comment by pallavg** [comment]
Could this issue be prioritized as it prevents django-wiki from working in a django 3.0 project? Thank you.

**Comment by matthiask** [comment]
I don't know whether anybody is working on this currently. I'd certainly appreciate a pull request @pallavg

**Comment by pallavg** [comment]
Hi @matthiask,

I had hacked the source code in the following files to get django-wiki to work on my Django 3.0.1 project. The main issue was django.utils.six ModuleError.

```
models.py:
#from django.utils import six
import six
```

```
utils.py:
#from django.utils.six import next, PY3, text_type
#from django.utils.six.moves import zip
from six import next, PY3, text_type
from six.moves import zip
```

I looked at the master branch and it seems like these imports have been totally removed. So I am not sure if there is anything further to be done as far django.utils.six is concerned.

**Comment by claudep** [comment]
No, the main issue is not related to `six` issues, it is related to the fact that in Django 3.0, setting a value to an `instance.foreign_id` field is automatically setting the `instance.foreign` value. django-mptt was using the capability to have temporary non-matching `instance.foreign_id`/`instance.foreign`, which is not possible any more.

**Comment by slinkymanbyday** [comment]
apologies for the spam, but is there a chance of a new release as this has been merged in?

**Comment by matthiask** [comment]
Closed in #724
